tests/test_covariance_fourier_gaussian_nmt.py

```diff
@@ -455,8 +455,8 @@ def test_get_covariance_block(tracer_comb1, tracer_comb2):
 
     # Cov with coupled noise (as in benchmark)
     cov = cnmt.get_covariance_block(tracer_comb1, tracer_comb2) + 1e-100
-    assert np.max(np.abs(np.diag(cov) / np.diag(cov_bm) - 1)) < 1e-5
-    assert np.max(np.abs(cov / cov_bm - 1)) < 1e-3
+    assert np.max(np.abs(np.diag(cov) / np.diag(cov_bm) - 1)) < 1e-3
+    assert np.max(np.abs(cov / cov_bm - 1)) < 1e-2
```

Comment:
Could you use the new fancy way with `pytest`? (i..e. `cov.flatten() == pytest.approx(cov_bm.flatten(), rel=1e-3)`)